skins/oasis/css/core/ads.scss

```diff
@@ -120,7 +120,7 @@ $video-interactive-area-z-index: $vpaid-z-index + 1;
 
 .wikia-ad {
 	&.slot-animation {
-		@include transition(max-height, 0.8s, ease, 0);
+		@include transition(max-height, 1.6s, ease, 0);
```

Comment:
Fixed line 15.

@nandy-andy can you point what is wrong in line 14? This file contains only variables so there is nothing that could be leaked to output css file.